fe/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -585,6 +585,11 @@ private void handleInsertStmt() throws Exception {
         } else {
             insertStmt = (InsertStmt) parsedStmt;
         }
+
+        // assign request_id
```

Comment:
Better to comment why you assign query id here.

Reply:
OK, done.